includes/bugzilla-date.js

```diff
@@ -65,6 +65,9 @@ function prettydate(d) {
         weeks = 0,
         months = 0;
 
+    if (diff < -600) {
+        return 'Future!'; // Found mistakes ASAP
```

Comment:
I don't know, I just hope that more users find and report errors. Users can still turn off this feature ("Turn timestamps into relative dates") in settings. I think it should not fail silently.